src/index.html

```diff
@@ -6,9 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="/images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#home" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#home"
+            class="nav__link" data-qa="hover">Laptops & Computers</a>
+          </li>
```

Comment:
apply code styling recommendations from a checklist
```suggestion
          <li class="nav__item">
            <a 
              href="#home"
              class="nav__link" 
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
```

Reply:
✅